src/components/Avatar/Avatar.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import './Avatar.css';
+
+export type AvatarProps = {
+  src?: string;
+  alt?: string;
+  letter?: string;
+};
+
+const Avatar: React.FC<AvatarProps> = ({ src, alt, letter }) => {
+  if (!src) {
+    return (
+      <div
+        className="avatar"
+        style={{ backgroundColor: 'var(--accent-color)' }}
```

Comment:
можно вынести в какой-то дополнительный класс и использовать как модификатор
```suggestion
        className="avatar avatar_accent"
```